demos/pedestrian_tracker_demo/cpp/main.cpp

```diff
@@ -126,14 +143,46 @@ int main(int argc, char **argv) {
 
         bool should_save_det_log = !detlog_out.empty();
 
+        //model API part
+        std::vector<std::string> labels;
+        if (!FLAGS_labels.empty())
+            labels = DetectionModel::loadLabels(FLAGS_labels);
+
+        std::unique_ptr<ModelBase> detection_model;
+        if (FLAGS_at == "centernet") {
+            detection_model.reset(new ModelCenterNet(det_model, (float)FLAGS_t, labels));
+        }
+        else if (FLAGS_at == "faceboxes") {
+            detection_model.reset(new ModelFaceBoxes(det_model, (float)FLAGS_t, FLAGS_auto_resize, (float)FLAGS_iou_t));
+        }
+        else if (FLAGS_at == "retinaface") {
+            detection_model.reset(new ModelRetinaFace(det_model, (float)FLAGS_t, FLAGS_auto_resize, (float)FLAGS_iou_t));
+        }
+        else if (FLAGS_at == "retinaface-pytorch") {
+            detection_model.reset(new ModelRetinaFacePT(det_model, (float)FLAGS_t, FLAGS_auto_resize, (float)FLAGS_iou_t));
+        }
+        else if (FLAGS_at == "ssd") {
+            detection_model.reset(new ModelSSD(det_model, (float)FLAGS_t, FLAGS_auto_resize, labels));
+        }
+        else if (FLAGS_at == "yolo") {
+            detection_model.reset(new ModelYolo(det_model, (float)FLAGS_t, FLAGS_auto_resize, FLAGS_yolo_af, (float)FLAGS_iou_t, labels));
+        }
+        else {
+            slog::err << "No model type or invalid model type (-at) provided: " + FLAGS_at << slog::endl;
+            return -1;
+        }
```

Comment:
Part of this networks don't detect persons (faceboxes, retinaface, retinaface-pytorch), you should remove them, and for those that detect filter boxes with ID corresponds to persons (centernet, ssd, yolo networks).

Reply:
@VoronovaIntern please make comments if you have addressed questions from review, otherwise it is not clear if you agreed to make changes or not agreed, or just missed/ignored comment

Reply:
@ivikhrev @vladimir-dudnik  I've done it